backend/init_data/skills/dingtalk-knowledge/provider.py

```
@@ -0,0 +1,650 @@
+
+            # Get or create sandbox
+            sandbox, error = await sandbox_manager.get_or_create_sandbox(
+                shell_type="ClaudeCode",
+                workspace_ref=None,
+            )
+
+            if error:
+                raise RuntimeError(f"Failed to create sandbox: {error}")
+
+            # Prepare paths
+            temp_dir = f"/tmp/dingtalk_{int(time.time())}"
```

Ruff (0.15.10)

[error] Probable insecure usage of temporary file or directory: "/tmp/dingtalk_"

(S108)